### PR TITLE
Remove min width

### PR DESCRIPTION
**Pull request by rokburgar**
- remove min size from floater
- add option to open over the input field

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -318,6 +318,19 @@
                 <option value="10">Ten</option>
             </select>
 
+            <select class="reform-selectbox reform-pure" multiple title="Open over field" data-over="true" >
```
Comment (gregorpraprotnik) [inline demo/index.html]: Please think of a more descriptive name for `over`.

File: less/floater_abstract.less [hunk 2]
```diff
@@ -14,7 +14,6 @@
     .rounded-corners;
 
     .reform-floater-list {
-        min-width  : 180px;
```
Comment (gregorpraprotnik) [inline less/floater_abstract.less]: Are you sure this doesn't cause any regressions?

File: src/selectbox_abstract.coffee [hunk 3]
```diff
@@ -286,16 +286,19 @@ class SelectBoxAbstract
...
-                    posTopAfterAnimation = pos.top - parseInt @orig.data('shift') 
-                    pos.top -= 1
-            else
-                pos.top = pos.top + @fake.outerHeight()
-                if @orig.data 'shift'
-                    posTopAfterAnimation = pos.top + parseInt @orig.data('shift') 
-                    pos.top += 1
+
+            # if over is set floater will open over the input not bellow
+            if not @options.over
```
Comment (gregorpraprotnik) [inline src/selectbox_abstract.coffee]: `unless @options.over`

File: less/floater_abstract.less (hunk 2 above)
Comment (rokburgar) [inline less/floater_abstract.less]: Reverting this. Will override it in the preview.

**Pull request merged**